subsys/mgmt/mcumgr/lib/cmd/img_mgmt/src/img_mgmt_util.c

```diff
@@ -8,7 +8,7 @@
 #include <string.h>
 
 #include "img_mgmt/image.h"
-#include "img_mgmt/img_mgmt.h"
+#include "img_mgmt/image_info.h"
```

Comment:
This creates 2 sets of file names `img_* and `image_*` it would be better to stick with one

Reply:
The naming image_state and image_info has been selected for that reason: these are not img_mgmt functions at all.
Registration functions are, callback are but these functions are img_mgmt independent.
Image info functions work on image directly and state functions are not img_mgmt_state functions,, if you look at "image list" that is image state command, these set image state as defined by MCUboot, and work on image directly - they do not require mcumgr at all.